docs/data-sources/advanced_cluster.md

```diff
@@ -124,8 +124,6 @@ To learn more, see [Resource Tags](https://dochub.mongodb.org/core/add-cluster-t
 
 ### labels
 
-**WARNING:** This property is deprecated and will be removed in the future, use the `tags` attribute instead.
```

Comment:
should we add changelog entries under NOTES section to clarify this change?

Reply:
Not sure. WDYT @Zuhairahmed ?

Reply:
no i would rather not add anything to changelog here. reason is because in a few months we will be adding this deprecation warning back once upstream team has finalized a new EOL date